### PR TITLE
Removed System.out command from parsePluginYamlFileTest

### PR DESCRIPTION
**Pull request by mikepain**
#103 & #122 as discussed with @timja I have created a new pull request with the one line of code removed

### CODE REVIEW

File: plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java [hunk 1]
```diff
@@ -161,11 +161,20 @@ public void start(boolean downloadUc) {
         System.out.println("Done");
     }
 
+    /**
+     * Create plugin directory
+     */
     void createRefDir() {
+
+        if ((refDir == null) ||
```
Comment (timja) [inline plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java:169]: I think these changes were meant for a different PR?
you should create a new branch for this and create a PR from there

**Comment by mikepain** [comment]
Sorry

I obviously misunderstand how I should use forks. I thought this change
would be isolated from the work I created a pull request for yesterday .

I thought I could make all changes on the master of that fork. But instead
I should  make all changes on  a branch of that fork.

Is that the correct way to do it?

I will take a branch.

Thanks

Mike

**Comment by timja** [comment]
Yes you should never work on the master branch of a fork as you'll hit issues like this.

Create a branch for each pull request.

**Comment by mikepain** [comment]
Ok understood, I thought as it was my fork I could work on my master I
could commit &  it would be separate from the Jenkins master

**Comment by mikepain** [comment]
OK will also remove the changes from my fork master.

**Comment by jetersen** [comment]
@mikepain perhaps https://lab.github.com/ is a good place to start :)

This doc [creating a pull request from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) actually mention the word topic branch 👍

**Comment by mikepain** [comment]
> @mikepain perhaps https://lab.github.com/ is a good place to start :)
> 
> This doc [creating a pull request from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) actually mention the word topic branch 👍

Thanks @jetersen

**Comment by mikepain** [comment]
> > @mikepain perhaps https://lab.github.com/ is a good place to start :)
> > This doc [creating a pull request from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) actually mention the word topic branch 👍
> 
> Thanks @jetersen

Thanks @jetersen , as suggested I went through some of the labs & looked at the Github documentation & feel more confident in using Github.

I now understand why the project uses forks, I thought they were instead of branches but I see they should be used as well a branches.

How should I clear up the problem I have created, as my master now contains the fixes for issues #150 & #97?
Should I do the following:

1. Rollback my fork master to the current upstream master
2. Create a fork branch & then fix both issues on that branch
3. Then use on pull request for both issues?

Thanks for your help

Mike

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
This pull request can be closed.  The change that was to be the original purpose has been made.

**Pull request closed without merging**